resources/migrations/20170222203502-aggregates.up.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE aggregates (
+  id          UUID PRIMARY KEY NOT NULL DEFAULT gen_random_uuid(),
+  description TEXT             NOT NULL,
+  activities  UUID REFERENCES activities (id)
```

Comment:
Correct. A single aggregate is a combination of multiple activities. Hence an array type :D